stix_shifter_modules/qradar/stix_translation/json/to_stix_map.json

```diff
@@ -4,9 +4,17 @@
     "cybox": false,
     "transformer": "ToInteger"
   },
-  "username": {
-    "key": "user-account.user_id"
-  },
+  "username": [
+    {
+      "key": "user-account.user_id",
+      "object": "user"
+    },
+    {
+      "key": "process.creator_user_ref",
+      "object": "process_image",
+      "references": "user"
```

Comment:
nothing wrong with the mapping. but its not making any sense. there's no process object mapping and yet user reference is created.

Reply:
makes sense, this is because the rest of the process fields are still missing for qradar - I will add them to this PR